build.gradle

```diff
@@ -705,6 +705,8 @@ uiTestTargets.each { version ->
     systemProperty 'ui-test.intellij.version', version
     systemProperty 'ui-test.plugin.path', buildPlugin.outputs.files[0].path
 
+    outputs.upToDateWhen { false }
```

Comment:
To always re-run the tests... even in local setup

For some reason, CI doesn't seem to re-use the cached test results anyway 🤔